exporter/kinesisexporter/otlp_marshaller.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kinesisexporter
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+type otlpProtoMarshaller struct{}
+
+var _ Marshaller = (*otlpProtoMarshaller)(nil)
+
+func (*otlpProtoMarshaller) MarshalTraces(traces pdata.Traces) ([]Message, error) {
```

Comment:
Currently this function marshals the whole batch as one payload. This is what we expect internally, however it is inconsistent with the jaeger marshaller which returns byte data per trace.

We could add individual trace encoding if required.

Reply:
I think it's OK as long as it's is documented properly.